### PR TITLE
mvenherak. OOP: Parking Lot, 1st edition

### PR DESCRIPTION
**Pull request by venherak**

### CODE REVIEW

File: src/main/java/parkinglot/ParkingLot.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            parkingSpots[i] = new ParkingSpot(i + 1);
+            freeSpots++;
+        }
+    }
+
+    private boolean isFull () {
+        if (freeSpots == 0) {
+            return true;
+        }
+        return false;
```
Comment (olobanok) [inline src/main/java/parkinglot/ParkingLot.java]: you may simply return freeSpots == 0;
result is the same but less code

File: src/main/java/parkinglot/ParkingLot.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            return  display("Add", freeParkingSpot);
+        }
+    }
+
+    public String removeVehicleFromParkingLot(Vehicle vehicle, String parkingTicketNumber) {
+        freeSpots++;
+        int spot = -1;
+
+        for (int i = 0; i < capacity; i++) {
+            if (parkingTickets[i].getParkingTicketNumber() == parkingTicketNumber) {
```
Comment (olobanok) [inline src/main/java/parkinglot/ParkingLot.java]: you should compare strings using equals(), not ==

File: src/main/java/parkinglot/ParkingLot.java (hunk 1 above)
Comment (venherak) [reply src/main/java/parkinglot/ParkingLot.java]: I agree. Will be corrected in next task

File: src/main/java/parkinglot/ParkingLot.java (hunk 2 above)
Comment (venherak) [reply src/main/java/parkinglot/ParkingLot.java]: I agree. Will be corrected in next task

File: src/main/java/parkinglot/ParkingLot.java (hunk 1 above)
Comment (venherak) [reply src/main/java/parkinglot/ParkingLot.java]: Corrected. https://github.com/AqaCoreSchool/homeworks/pull/28/commits/57099748843c75dfa4f31fcca940318807f1aa55

File: src/main/java/parkinglot/ParkingLot.java (hunk 2 above)
Comment (venherak) [reply src/main/java/parkinglot/ParkingLot.java]: Corrected. https://github.com/AqaCoreSchool/homeworks/pull/28/commits/57099748843c75dfa4f31fcca940318807f1aa55

**Comment by nmelnyk-corevalue** [comment]
good!

**Pull request closed without merging**